site/layouts/shortcodes/ods-alert.html

```diff
@@ -0,0 +1,21 @@
+{{- /*
+Usage: `ods-alert "level"`, where `level` is one of info, success, warning or danger (default)
+*/ -}}
+
+{{- $level := .Get 0 | default "danger" -}}
+
+<div class="alert alert-{{ $level }}">
+  <span class="alert-icon"><span class="visually-hidden">{{ $level }}</span></span>
+  <div>
+    <div class="alert-heading mb-1 h4">
```

Comment:
With `.h4` the rendering is different compared to our _main_ branch. Why do we need this change? Does it come from a specification/guideline or a discussion with the designers?